third_party/thrift/thrift/Thrift.h

```diff
@@ -38,7 +38,7 @@
 #include <map>
 #include <list>
 #include <set>
-#include <vector>
```

Comment:
This I am not sure we want to touch third party stuff, it's not guaranteed to actually work